### DISCUSSION TITLE
add logo in gallery

### DISCUSSION THREAD

**Discussion by SRNV** [nestdotland/nest.land#235] [Ideas]
just to identify quickly

**Comment by t8** [comment]
Are you referring to custom module logos or the nest.land logo?

**Comment by SRNV** [comment]
Custom

**Comment by zorbyte** [comment]
Interesting idea. We could setup this up in our new configuration system (CLI) for people to either use a URL (which we then upload to the arweave) or upload their own (also uploaded to the Arweave)

**Reply by t8** [reply]
We could, but we'd need a place for this in the design. I'm worried that adding logos will clutter certain things. Besides, for organization accounts (coming soon!) the profile image would most likely be the "logo."

CC @maximousblk
